osf/models/node.py

```diff
@@ -729,7 +729,7 @@ def update_search(self):
 
         try:
             search.search.update_node(self, bulk=False, async_update=True)
-            if self.is_collected and self.is_public:
+            if self.collection_submissions.exists() and self.is_public:
```

Comment:
why? We still have the `is_collected` property

Reply:
It's just used in tests basically, removed!